src/resmom/linux/mom_mach.c

```diff
@@ -193,11 +193,6 @@ typedef	struct	proc_mem  {
 } proc_mem_t;
 
 int	mom_does_chkpnt = 0;
-ulong	cpuset_nodes = 1;
-ulong	memreserved = 0;
-ulong	mempernode;
```

Comment:
You need to remove this (and possibly other) external definitions that are present in mom_mach.h.

Reply:
@mike0042 There don't seem to be other references to those in mom_mach.h

Reply:
Yes, that appears to be what happened. LGTM. I'll leave my blocking review in place to sync with https://github.com/PBSPro/pbspro/pull/1710